demos/README.md

```diff
@@ -90,6 +90,20 @@ file describes the demo scenarios.
 
 6. Replace all occurrences of the k8ssandra-operator namespace. At the time of this writing, the areas to search and replace include the namespace field for a given resource in addition to the cert-manager annotations (cert-manager.io/inject-ca-from). Note: there may be kustomize variables used for the cert-manager.io/inject-ca-from annotation, so ensure all usages reference the px-k8ssandra namespace.
 
+## Updating the `px-mongo` demo
+
+1. Clone `https://github.com/kpattaswamy/mern-k8s`
```

Comment:
We probably don't want to link to a personal Github repo. We should probably pull this into one of Pixie's repos. 
What sorts of changes are on this fork?

Reply:
The upstream demo app was done by the MongoDB team and relied on the demo's backend interacting with their hosted MongoDB atlas instance.

My fork made changes to that demo to create a local MongoDB instance and let the backend service interact with that instead of MongoDB's hosted one. It also contains the necessary docker compose/kubernetes manifests to run the demo locally and utilizes locust for constant load on the various backend endpoints.

Reply:
Agree with @aimichelle. We shouldn't linked to personal forks. If we need a fork, it should be under the pixie-io org.

Reply:
Seems like the fork has been created and upstream added a license. Is there anything else to do here?